### DISCUSSION TITLE
Support for PDF/A-3B

### DISCUSSION THREAD

**Discussion by isential-gmbh** [QuestPDF/QuestPDF#1044] [Q&A] [answered]
Hello,

Can PDF/A-3B be specified as a target format?

Thank you and best regards,
René

**Comment by MarcinZiabek** [comment, marked as answer]
The PDF/A-3b support has just been released in the 2024.10.3 version. Could you please try it?

**Reply by isential-gmbh** [reply]
Thanks, and sure, but only when I get the chance. I've been assigned a project that I need to complete first.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!

**Reply by isential-gmbh** [reply]
At the moment, I don't have the time to do full and in-depth testing. But I will get to it as soon as my workload allows and will report back here.
Thanks and best regards,
René
